std_msgs/Header header
uint16 status
uint16 sample_counter
float32 pitch
float32 roll
float32 yaw

